src/pymorize/data/dimensionless_mappings.yaml

```diff
@@ -5,3 +5,5 @@ so:
   "0.001": g/kg
 sos:
   "0.001": g/kg
```

Comment:
is it possible to avoid duplicated values?

Reply:
No